docs/en/latest/practices/how-to-use-go-plugin-runner-in-apisix-ingress.md

```diff
@@ -0,0 +1,400 @@
+# How to use go-plugin-runner in APISIX Ingress
+
+## Background Description
+
+While wandering around the community, I found a user confused about "how to use multilingual plugins in APISIX Ingress environment". I happen to be a user of go-plugin-runner and have a little knowledge of the APISIX Ingress project, so this document was born.
+
+## Proposal Description
+
+Based on version 0.3 of the go-plugin-runner plugin and version 1.4.0 of APISIX Ingress, this article goes through building the cluster, building the image, customizing the helm chart package, and finally, deploying the resources. It is guaranteed that the final result can be derived in full based on this documentation.
+
+```bash
+go-plugin-runner: 0.3
+APISIX Ingress: 1.4.0
+
+kind: kind v0.12.0 go1.17.8 linux/amd64
+kubectl version: Client Version: v1.23.5/Server Version: v1.23.4
+golang: go1.18 linux/amd64
+```
+
+## Begin
+
+### Build a cluster environment
+
+Select `kind` to build a local cluster environment. The command is as follows:
+
+```bash
+cat <<EOF | kind create cluster --config=-
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+nodes:
+- role: control-plane
+  kubeadmConfigPatches:
+  - |
+    kind: InitConfiguration
+    nodeRegistration:
+      kubeletExtraArgs:
+        node-labels: "ingress-ready=true"
```

Comment:
it's not necessary

Reply:
The following installation commands?

Reply:
waiting for more suggestions.

Reply:
@tao12345666333 Why is it not necessary?

Reply:
line 32～37

Reply:
Even without this configuration line 32～37 , APISIX Ingress can be installed and run normally.

Introducing extra content can cause confusion

Reply:
@tao12345666333  Got it~
I am testing this process.

Reply:
Test successfully~

Reply:
Solved~